db/db_iter.h

```diff
@@ -476,8 +507,11 @@ class DBIter final : public Iterator {
   std::string saved_timestamp_;
   std::optional<std::vector<ScanOptions>> scan_opts_;
   ReadOnlyMemTable* const active_mem_;
-  const SequenceNumber memtable_seqno_lb_;
-  const uint32_t memtable_op_scan_flush_trigger_;
+  SequenceNumber memtable_seqno_lb_ = kMaxSequenceNumber;
+  uint32_t memtable_op_scan_flush_trigger_ = 0;
+  uint32_t avg_op_scan_flush_trigger_ = 0;
+  uint32_t iter_step_since_seek_ = 1;
+  uint32_t mem_hidden_op_scanned_since_seek_ = 0;
```

Comment:
Sorry - for some reason my comment here is not saved. Is it possible to use consistent initialization (either inline or during constructor)?

Reply:
Piggybacked in https://github.com/facebook/rocksdb/pull/13632/commits/467fa18277e1fc55457c9aef428391e7ed188ee9